native/cocos/application/BaseGame.cpp

```diff
@@ -46,12 +48,23 @@ int BaseGame::init() {
                                                       cc::ISystemWindow::CC_WINDOW_INPUT_FOCUS
                                                 : _windowInfo.flags;
     std::call_once(_windowCreateFlag, [&]() {
-        if (_windowInfo.x == -1 || _windowInfo.y == -1) {
-            createWindow(_windowInfo.title.c_str(), _windowInfo.width, _windowInfo.height, _windowInfo.flags);
-        } else {
-            createWindow(_windowInfo.title.c_str(),
-                         _windowInfo.x, _windowInfo.y, _windowInfo.width, _windowInfo.height, _windowInfo.flags);
-        }
+        //if (_windowInfo.x == -1 || _windowInfo.y == -1) {
+        //    createWindow(_windowInfo.title.c_str(), _windowInfo.width, _windowInfo.height, _windowInfo.flags);
+        //} else {
+        //    createWindow(_windowInfo.title.c_str(),
+        //                 _windowInfo.x, _windowInfo.y, _windowInfo.width, _windowInfo.height, _windowInfo.flags);
+        //}
+
+        ISystemWindowInfo info;
+        info.title = _windowInfo.title;
+        info.x = _windowInfo.x == -1 ? 0 : _windowInfo.x;
+        info.y = _windowInfo.y == -1 ? 0 : _windowInfo.y;
+        info.width = _windowInfo.width;
+        info.height = _windowInfo.height;
+        info.flags = _windowInfo.flags;
+
+        ISystemWindowManager* windowMgr = CC_GET_PLATFORM_INTERFACE(ISystemWindowManager);
+        ISystemWindow* window1 = windowMgr->createWindow(info);
```

Comment:
Where is `window1` used?

Reply:
Not used for now

Reply:
Then what is it used for?

Reply:
Just `windowMgr->createWindow(info);` is OK.
I was thinking `window1` maybe used in future.

Reply:
I think only main window should be created automatically. Other windows should be created manually.

Reply:
And currently, it is useless and memory leak.